components/db/testimony/usePublishedTestimonyListing.ts

```diff
@@ -91,7 +100,21 @@ async function listTestimony(
   limitCount: number,
   startAfterKey: unknown | null
 ): Promise<Testimony[]> {
+  if (refinement.profilePage && refinement.uid) {
+    const client = createClient()
+
+    const data = await client
+      .collections("publishedTestimony")
+      .documents()
+      .search({ q: refinement.uid, query_by: "authorUid" })
+
+    return data.hits
+      ? data.hits.map(({ document }) => document as Testimony)
```

Comment:
These documents actually have type TestimonySearchRecord, which we should update this to return.

Reply:
`import type { TestimonySearchRecord } from '../../functions/src/testimony/types'`